packages/broker/src/ConfigWizard.ts

```diff
@@ -260,11 +262,11 @@ export const createStorageFile = async (config: any, answers: inquirer.Answers):
 
 export const startBrokerConfigWizard = async(): Promise<void> => {
     try {
-        const answers = await inquirer.prompt(prompts)
-        const config = getConfigFromAnswers(answers)
-        if (answers.revealGeneratedPrivateKey) {
-            logger.info(`This is your node\'s private key: ${config.ethereumPrivateKey}`)
-        }
+        let config = { ... CONFIG_TEMPLATE, plugins: { ... CONFIG_TEMPLATE.plugins } }
+        const ethereumAnswers = await inquirer.prompt(privateKeyPrompts)
+        config = getEthereumConfigFromAnswers(ethereumAnswers, config)
+        const pluginAnswers = await inquirer.prompt([pluginSelectorPrompt, ...pluginPrompts])
```

Comment:
Maybe the global `pluginPrompts` variable would include all plugin-related prompts. (If the initialization of that variable is not trivial, wrap the logic into a method.)